README.md

```diff
@@ -95,6 +95,9 @@ Yes, you will need to have WordPress 6.1 or later installed to take advantage of
 
 ## Changelog
 
+### 0.2.0
+* Add 'PM Pattern Block' to render a pattern in a pattern.
```

Comment:
```suggestion
* Add 'Pattern Block' to render a pattern in a pattern.
```

Reply:
Ah, good point!

Reply:
Committed in https://github.com/studiopress/pattern-manager/pull/161/commits/653f41aa2c98db58b41bae81dd7d5270c7c08611